src/DropDown/dropdown-style.scss

```diff
@@ -1,94 +1,313 @@
+$transition-time: 300ms;
+$icon-size: 18px;
+$dropdown-width: 100%;
+$dropdown-toggle-height: 44px;
+$blue: #41b0ee;
+$darkBlue1: #0092e1;
+$darkBlue2: #007ac7;
+$text: #444444;
+$grey6: #999999;
+$grey7: #b3b3b3;
```

Comment:
Please change these to match SEB style variables

Reply:
fixed this in commit https://github.com/sebgroup/react-components/pull/26/commits/92ad13f895d88dc53f26f638dff9cd47c82d9999